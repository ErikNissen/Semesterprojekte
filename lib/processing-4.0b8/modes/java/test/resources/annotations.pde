version https://git-lfs.github.com/spec/v1
oid sha256:ffefb0ed4b98db31eb360d7ef3bc79d09eb8a30c71cc2eeac4485e19683ff8f4
size 442
