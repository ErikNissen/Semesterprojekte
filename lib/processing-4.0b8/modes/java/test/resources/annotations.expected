version https://git-lfs.github.com/spec/v1
oid sha256:ab1df644afa4c5397ce5148106512a98b7fc610aca5d4a6fbdece9d31f966c17
size 1176
